src/customs/useCalculator.ts

```diff
@@ -0,0 +1,65 @@
+import { useState } from 'react'
+
+import { DigitNumber, Operation } from '@/types/calculator'
+
+import { calculator } from '@/utils/calculator'
+
+const useCalculator = () => {
+  const [total, setTotal] = useState(0)
+  const [operator, setOperator] = useState({
+    leftPort: 0,
+    rightPort: 0,
+    expression: '',
+  })
+
+  const handleDigit = (digit: DigitNumber) => {
+    const { expression } = operator
+    const newTotal = Boolean(expression) ? digit : Number(total.toString().concat(digit.toString()))
+    if (newTotal > 999) return alert('세 자리 수 이상 입력할 수 없습니다.')
```

Comment:
```suggestion
    if (newTotal > 999) {
      alert('세 자리 수 이상 입력할 수 없습니다.')
      return
    }
```

alert는 값을 반환하지 않을 뿐더러, 반환하는 값이 큰 의미가 없으니까, 
명시적으로 alert를 호출하고 return을 호출하는 것이 더 코드의 의미가 잘 전달될 것 같아요 😄